README.md

```diff
@@ -84,6 +84,7 @@ and a transform stream to decompress the file.
 | [options.check] | <code>Boolean</code> | <code>false</code> | enable write check |
```

Comment:
In which case I guess that ought to be mentioned in the README too (once implemented)  :-)